quote-book/tests/common/mod.rs

```diff
@@ -172,6 +176,31 @@ pub fn basic_happy_flow(quote_book: &impl QuoteBook) {
     );
 }
 
+/// Test adding a quote which is already in the ledger
+pub fn add_quote_already_in_ledger_should_fail(
```

Comment:
This is a specific test to the SynchronizedOrderBook implementation, so I think it can move into `quote-book/tests/synchronized_quote_book.rs`